pkg/commands/drift/flags/driftissue.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package flags
+
+import (
+	"github.com/abcxyz/pkg/cli"
+)
+
+// DriftIssueFlags represent the shared drift github issue flags among all commands.
+// Embed this struct into any commands that needs to create drift GitHub issues.
+type DriftIssueFlags struct {
+	FlagSkipGitHubIssue            bool
+	FlagGitHubIssueLabels          []string
+	FlagGitHubIssueAssignees       []string
+	FlagGitHubCommentMessageAppend string
+}
+
+func (d *DriftIssueFlags) AddFlags(set *cli.FlagSet, defaultIssueLabel string) {
```

Comment:
This API is a little bit weird. I wonder if we should accept a struct instead?

Reply:
Done